skimage/restoration/tests/test_rolling_ball.py

```diff
@@ -18,6 +19,15 @@ def test_ellipsoid_const():
     assert np.allclose(img - background, np.zeros_like(img))
 
 
+@testing.parametrize('dtype', [np.float32, np.float64])
```

Comment:
same here: is it worth testing all relevant dtypes?

Reply:
done